packages/ra-ui-materialui/src/detail/TabbedShowLayoutTabs.js

```diff
@@ -21,6 +21,7 @@ const TabbedShowLayoutTabs = ({ value, children, match }) => (
             return cloneElement(tab, {
                 context: 'header',
                 value: tabPath,
+                ...rest,
```

Comment:
Sorry, you misunderstood my comment, I wanted the rest props to be passed to `Tabs`, not to its children.

Reply:
oh, yeah, you said "`Tab`" and that got me confused haha. Well... should I keep on explicitly  passing `value={value}` in `Tabs` as well, or do I just remove it since it'll be in `rest`?